specification/components/stu3/schemas/endpoints/a020-upload-file-to-document-store.yaml

```diff
@@ -56,7 +56,6 @@ description: |
 summary: A020 - Upload file to document store
 operationId: a020-upload-file-to-document-store
 parameters:
-  - $ref: '../headers/request/ContentTypeAttachmentsUpload.yaml'
```

Comment:
Now that this parameter has been removed is the `ContentTypeAttachmentsUpload` component still used elsewhere? With the removal of this parameter is the need for a `content-type` header still documented?